lightning/index.js

```diff
@@ -2,9 +2,15 @@ const express = require('express');
 const fs = require('fs');
 const cors = require('cors');
 const app = express();
+const {MongoClient} = require('mongodb');
+const dotEnv = require('dotenv').config()
 app.use(express.json());
 app.use(cors());
 
+//MongoDB connection string
+const uri = "mongodb+srv://" + process.env.REACT_APP_MONGO_DB_ID + ":" + process.env.REACT_APP_MONGO_DB_SECRET + "@jobs.tcmdh.mongodb.net/jobs?retryWrites=true&w=majority"
```

Comment:
don't technically need the env variables with the `REACT_APP` prefix 

also consider using string template literals for readability